### PR TITLE
[15.0][MIG] helpdesk_mgmt_crm: Migration to 15.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
cc @Tecnativa TT50685

@victoralmau @juanjosesegui-tecnativa please review

### CODE REVIEW

**Review by juanjosesegui-tecnativa — APPROVED** [review]

File: helpdesk_mgmt_crm/security/ir.model.access.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+helpdesk_mgmt_crm.access_helpdesk_ticket_create_lead,access_helpdesk_ticket_create_lead,helpdesk_mgmt_crm.model_helpdesk_ticket_create_lead,base.group_user,1,0,0,0
+helpdesk_mgmt_crm_salesman.access_helpdesk_ticket_create_lead,access_helpdesk_ticket_create_lead,helpdesk_mgmt_crm.model_helpdesk_ticket_create_lead,sales_team.group_sale_salesman,1,1,1,1
```
Comment (pedrobaeza) [inline helpdesk_mgmt_crm/security/ir.model.access.csv]: Don't give by default read permissions to internal users, and unlink permission is not usually granted:

```suggestion
helpdesk_mgmt_crm_salesman.access_helpdesk_ticket_create_lead,access_helpdesk_ticket_create_lead,helpdesk_mgmt_crm.model_helpdesk_ticket_create_lead,sales_team.group_sale_salesman,1,1,1,0
```

File: helpdesk_mgmt_crm/wizard/helpdesk_ticket_create_lead.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        return vals
+
+    def _prepare_vals(self):
+        return {
+            "ticket_id": self.ticket_id.id,
+            "name": self.ticket_id.name,
+            "partner_id": self.ticket_id.partner_id.id,
+            "user_id": self.user_id.id or self.ticket_id.user_id.id,
+            "team_id": self.team_id.id,
+            "description": html2text.html2text(self.ticket_id.description),
```
Comment (pedrobaeza) [inline helpdesk_mgmt_crm/wizard/helpdesk_ticket_create_lead.py]: The field is now HTML, so no need of this. Please put attention of these things, specially as this implies an extra library.

```suggestion
            "description": self.ticket_id.description,
```

**Review by pedrobaeza — COMMENTED** [review]
/ocabot migration helpdesk_mgmt_crm

File: helpdesk_mgmt_crm/wizard/helpdesk_ticket_create_lead.py (hunk 2 above)
Comment (pilarvargas-tecnativa) [reply helpdesk_mgmt_crm/wizard/helpdesk_ticket_create_lead.py]: Thank you, I hadn't thought of that, I'll keep it in mind.

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+# generated from manifests external_dependencies
```
Comment (pedrobaeza) [inline requirements.txt]: Remove entirely this file

Comment (pilarvargas-tecnativa) [reply requirements.txt]: done!

**Comment by pedrobaeza** [comment]
@pilarvargas-tecnativa you have to squash administrative commits as well.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**